src/main/java/org/mskcc/cbio/portal/scripts/ImportTabDelimData.java

```diff
@@ -277,28 +294,71 @@ public void importData(int numLines) throws IOException, DaoException {
                 genericAssayStableIdToEntityIdMap = GenericAssayMetaUtils.buildGenericAssayStableIdToEntityIdMap();
             }
 
-            int lenParts = parts.length;
+            int headerColumns = headerParts.length;
 
             String line = buf.readLine();
             while (line != null) {
+
                 ProgressMonitor.incrementCurValue();
                 ConsoleUtil.showProgress();
                 boolean recordAdded = false;
 
-                // either parse line as geneset or gene for importing into 'genetic_alteration' table
-                if (isGsvaProfile) {
-                    recordAdded = parseGenesetLine(line, lenParts, sampleStartIndex, genesetIdIndex,
-                        filteredSampleIndices, daoGeneticAlteration);
-                } else if (isGenericAssayProfile) {
-                    recordAdded = parseGenericAssayLine(line, lenParts, sampleStartIndex, genericAssayIdIndex,
-                        filteredSampleIndices, daoGeneticAlteration, genericAssayStableIdToEntityIdMap);
-                } else {
-                    recordAdded = parseLine(line, lenParts, sampleStartIndex,
-                        hugoSymbolIndex, entrezGeneIdIndex, rppaGeneRefIndex,
-                        isRppaProfile, isDiscretizedCnaProfile,
-                        daoGene,
-                        filteredSampleIndices, orderedSampleList,
-                        existingCnaEvents);
+                if (FileUtil.isInfoLine(line)) {
```

Comment:
```suggestion
                if (FileUtil.isDataLine(line)) {
```
I would propose this name. Somehow "info line" to me sounds like the header metadata lines starting with `#` , e.g. I associate info with ℹ️

Reply:
Done https://github.com/cBioPortal/cbioportal-core/commit/f3d76c74e2da49c29de22cc19f78bfe2c11c05a1